plugins/fe_events_direct/externallib.php

```diff
@@ -0,0 +1,93 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Plugin version and other metadata.
+ *
+ * @package    local_fe_events_direct
+ * @copyright  2021 OpenStax
+ * @license    https://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+defined('MOODLE_INTERNAL') || die();
+require_once($CFG->libdir . '/externallib.php');
+require_once($CFG->libdir . "/filelib.php");
+
+/**
+ * Front End Events Direct Web Service
+ *
+ * @package    local_fe_events_direct
+ * @copyright  2021 OpenStax
+ * @license    https://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class local_fe_events_direct_external extends external_api {
+
+    /**
+     * Returns description of process_content_loaded_event() parameters
+     *
+     * @return external_function_parameters
+     */
+    public static function process_content_loaded_event_parameters() {
+        return new external_function_parameters(
+            array(
+                  'contentId' => new external_value(PARAM_TEXT, 'Content ID')                )
+            );
+    }
+
+    /**
+     * Forward content_loaded event to EventsAPI.
+     *
+     * @param str $contentid
+     * @return array empty array 
```

Comment:
Super small: It looks like there's an extraneous white space at the end of this line that the linter is griping about.